weed/iamapi/iamapi_management_handlers.go

```diff
@@ -212,37 +216,47 @@ func (iama *IamApiServer) CreatePolicy(s3cfg *iam_pb.S3ApiConfiguration, values
 	policyLock.Lock()
 	defer policyLock.Unlock()
 	if err = iama.s3ApiConfig.GetPolicies(&policies); err != nil {
-		return resp, err
+		return resp, &IamError{Code: iam.ErrCodeServiceFailureException, Error: err}
 	}
 	policies.Policies[policyName] = policyDocument
 	if err = iama.s3ApiConfig.PutPolicies(&policies); err != nil {
-		return resp, err
+		return resp, &IamError{Code: iam.ErrCodeServiceFailureException, Error: err}
```

Comment:
But in this case, we can't load the config from the s3 container, so I decided it's better to respons as `iam.ErrCodeServiceFailureException` --> `HTTP 500`